### PR TITLE
Fonts 2021 sql

### PR DESCRIPTION
**Pull request by konfirmed**
Progress on #2143 

Fonts queries (draft):

- [x] Web Usage- Overall
- [x] Web Usage - By Country
- [x] Hosting service vs Self-host service
- [x] Services Used
- [x] Font-display
- [x] Formats
- [x] Popular Fonts
- [x] Variable Fonts - Adoption
- [x] Variable Fonts - Axes used
- [x] Color Fonts

### CODE REVIEW

**Comment by tunetheweb** [comment]
@konfirmed is this good to review no or still work in progress?

If it's still WIP can you switch it to a draft PR for now, and also include the list of metrics with those that have been done and those that are still to do, so we can get a sense of how far through this is?

See some of the other PRs as examples:

image

Also FYI @jessthebp as think she was planning on looking at these queries too.

**Comment by konfirmed** [comment]
It is still work in progress, will revert. Will reach out to Jess. Thanks.

**Comment by tunetheweb** [comment]
Well happy for you to reopen this and leave it as draft. As I say we actually like that cause we can see work is being done - which is good for us to avoid panicking that no ones working on a chapters analysis!! 😁

Once it’s ready for review you can then take it out of draft.

**Comment by rviscomi** [comment]
Reopened and set to draft. Thanks for kicking this off @konfirmed!

**Comment by rviscomi** [comment]
@konfirmed @jessthebp is there anything I can do to help unblock this PR?

File: sql/2021/fonts/fonts_format.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+SELECT
+  client,
+  LOWER(IFNULL(REGEXP_EXTRACT(mimeType, '/(?:x-)?(?:font-)?(.*)'), ext)) AS mime_type,
+  COUNT(0) AS freq,
+  SUM(COUNT(0)) OVER (PARTITION BY client) AS total,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client) AS pct
+FROM
+  `httparchive.almanac.requests`
+WHERE
+  type = 'font' AND mimeType != '' AND date = '2021-07-01'
```
Comment (rviscomi) [inline sql/2021/fonts/fonts_format.sql:12]:
```suggestion
  date = '2021-07-01' AND
  type = 'font' AND
  mimeType != ''
```

File: sql/2021/fonts/fonts_format.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+GROUP BY
+  client,
+  mime_type
+ORDER BY
+  client, pct DESC
```
Comment (rviscomi) [inline sql/2021/fonts/fonts_format.sql:17]:
```suggestion
  client,
  pct DESC
```

File: sql/2021/fonts/variable_font_axes_used.sql [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  SUM(COUNT(DISTINCT page)) OVER (PARTITION BY client) AS total,
+  COUNT(DISTINCT page) / SUM(COUNT(DISTINCT page)) OVER (PARTITION BY client) AS pct
+FROM
+  `httparchive.almanac.parsed_css`,
+  UNNEST(getFontVariationSettings(css)) AS value,
+  UNNEST(SPLIT(value, ',')) AS values
+WHERE
+  date = '2021-07-01'
+GROUP BY
+  client, axis, num_axis
```
Comment (rviscomi) [inline sql/2021/fonts/variable_font_axes_used.sql:35]:
```suggestion
  client,
  axis,
  num_axis
```

**Review by rviscomi — CHANGES_REQUESTED** [review]
Just a few nits, thanks for working on these @konfirmed!

**Review by rviscomi — APPROVED** [review]

**Comment by rviscomi** [comment]
Merging 🚀

If there are any additional queries needed for this chapter, please reopen a new PR.

**Pull request merged**